<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>752</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="MainSideNavFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="MainSideNav">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="MainStackView">
      <widget class="QWidget" name="page1"/>
      <widget class="QWidget" name="page0"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuPlugins">
     <property name="title">
      <string>Plugins</string>
     </property>
     <property name="icon">
      <iconset resource="../../../source.qrc">
       <normaloff>:/icons/plugin-2.svg</normaloff>:/icons/plugin-2.svg</iconset>
     </property>
     <addaction name="actionRegister"/>
     <addaction name="actionList"/>
    </widget>
    <widget class="QMenu" name="menuData">
     <property name="title">
      <string>Data</string>
     </property>
     <property name="icon">
      <iconset resource="../../../source.qrc">
       <normaloff>:/icons/data.svg</normaloff>:/icons/data.svg</iconset>
     </property>
     <addaction name="actionImport"/>
     <addaction name="actionExport"/>
    </widget>
    <addaction name="menuPlugins"/>
    <addaction name="menuData"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPrefrences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContact"/>
    <addaction name="actionHelp"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionContact">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/contact-phonebook-support.svg</normaloff>:/icons/contact-phonebook-support.svg</iconset>
   </property>
   <property name="text">
    <string>Contact</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/info-square.svg</normaloff>:/icons/info-square.svg</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="shortcut">
    <string>Alt+F1</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/register.svg</normaloff>:/icons/register.svg</iconset>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="actionList">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/list.svg</normaloff>:/icons/list.svg</iconset>
   </property>
   <property name="text">
    <string>List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/import.svg</normaloff>:/icons/import.svg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/export.svg</normaloff>:/icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPrefrences">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/preferences.svg</normaloff>:/icons/preferences.svg</iconset>
   </property>
   <property name="text">
    <string>Prefrences</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../../source.qrc">
     <normaloff>:/icons/help.svg</normaloff>:/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../source.qrc"/>
 </resources>
 <connections/>
</ui>
